packages/lib-utils/src/components/details-page-header/utils/ActionButtons.tsx

```diff
@@ -3,41 +3,57 @@ import * as _ from 'lodash-es';
 import React from 'react';
 
 export type ActionButtonProp = {
-  label: string;
+  id?: string;
+  label?: string;
   callback: (event: React.MouseEvent) => void;
   isDisabled?: boolean;
   tooltip?: string;
+  variant?: 'primary' | 'secondary' | 'tertiary' | 'link';
```

Comment:
TODO (not blocking) can we forward the Patternfly values here instead of hard-coding it on our end? It seems you cannot get a list today it seems.

Reply:
Quick test locally - looks like you can import the ButtonVariant type from PF: `import type { ButtonVariant } from '@patternfly/react-core';` 

We should investigate in the future how to dedupe all of these props with PF props - and consider moving the ActionButtons outside of the details-page-header folder (they could theoretically be consumed independently of the header).